### PR TITLE
Feat Add Google One Tap script

### PR DESCRIPTION
**Pull request by adriadoz**
Feat Add Google One Tap script loader

### CODE REVIEW

File: components/googleOneTap/loader/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@s-ui/sui-google-one-tap-loader",
```
Comment (midudev) [inline components/googleOneTap/loader/package.json:2]: `@s-ui/react-google-one-tap-loader`

Comment (adriadoz) [reply components/googleOneTap/loader/package.json:2]: thx!

**Review by rackso — APPROVED** [review]

**Review by emiliovillu — APPROVED** [review]

File: test/googleOneTap/loader/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import chai, {expect} from 'chai'
+import chaiDOM from 'chai-dom'
+import {render} from '@testing-library/react'
+
+chai.use(chaiDOM)
+
+describe('GoogleOneTapLoader', () => {
+  it('Render', () => {
+    render(<GoogleOneTapLoader />)
+    expect(true).to.be.eql(false)
```
Comment (nucliweb) [inline test/googleOneTap/loader/index.js:12]: 🙈

File: demo/googleOneTap/loader/playground [hunk 3]
```diff
@@ -0,0 +1 @@
+return (<GoogleOneTapLoader render={()=>null}/>)
```
Comment (nucliweb) [inline demo/googleOneTap/loader/playground:1]: `timeoutRender` is a required prop.

```suggestion
return (<GoogleOneTapLoader render={()=>null} timeoutRender={()=>null} />)
```

File: components/googleOneTap/loader/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+$ npm install @s-ui/sui-google-one-tap-loader
+```
+
+## Usage
+
+### Basic usage
+```js
+import GoogleOneTapLoader from '@s-ui/sui-google-one-tap-loader'
+
+return (<GoogleOneTapLoader render={() => 'You can use Google Accounts now!'}/>)
```
Comment (nucliweb) [inline components/googleOneTap/loader/README.md:19]: `timeoutRender` is a required prop.
```suggestion
return (<GoogleOneTapLoader render={() => 'You can use Google Accounts now!'} timeoutRender={()=>null}/>)
```

**Review by nucliweb — APPROVED** [review]

File: demo/googleOneTap/loader/playground (hunk 3 above)
Comment (adriadoz) [reply demo/googleOneTap/loader/playground:1]: not required

File: components/googleOneTap/loader/README.md (hunk 4 above)
Comment (adriadoz) [reply components/googleOneTap/loader/README.md:19]: not required anymore, thanks!

File: components/googleOneTap/loader/src/index.js [hunk 5]
```diff
@@ -28,7 +28,7 @@ GoogleOneTapLoader.displayName = 'GoogleOneTapLoader'
 
 GoogleOneTapLoader.propTypes = {
   render: PropTypes.func.isRequired,
-  timeoutRender: PropTypes.func.isRequired,
+  timeoutRender: PropTypes.func,
```
Comment (nucliweb) [inline components/googleOneTap/loader/src/index.js:31]: 👍🏼

**Pull request merged**